File: dessert_life/src/main/java/project/shen/dessert_life/task/Utils/DebugLog.java
package project.shen.dessert_life.task.Utils;

import android.util.Log;

public class DebugLog {
    public static boolean isDebug = true;

    public static void logE(String tag, String value) {
        if (!isDebug) {
            return;
        } else {
            Log.e(tag, value);
        }
    }
    public static void logD(String tag, String value) {
        if (!isDebug) {
            return;

        } else {
            Log.d(tag, value);
        }

    }
    public static void logI(String tag, String value) {
        if (!isDebug) {
            return;
        } else {
            Log.i(tag, value);
        }

    }

    public static void logW(String tag, String value) {
        if (!isDebug) {
            return;
        } else {
            Log.w(tag, value);
        }
    }


    public static void logE(String tag, int value) {
        if (!isDebug) {
            return;
        } else {
            Log.e(tag, String.valueOf(value));
        }
    }
    public static void logD(String tag, int value) {
        if (!isDebug) {
            return;

        } else {
            Log.d(tag, String.valueOf(value));
        }

    }
    public static void logI(String tag, int value) {
        if (!isDebug) {
            return;
        } else {
            Log.i(tag, String.valueOf(value));
        }

    }

    public static void logW(String tag, int value) {
        if (!isDebug) {
            return;
        } else {
            Log.w(tag, String.valueOf(value));
        }
    }


    public static void logE(String tag, long value) {
        if (!isDebug) {
            return;
        } else {
            Log.e(tag, String.valueOf(value));
        }
    }
    public static void logD(String tag, long value) {
        if (!isDebug) {
            return;

        } else {
            Log.d(tag, String.valueOf(value));
        }

    }
    public static void logI(String tag, long value) {
        if (!isDebug) {
            return;
        } else {
            Log.i(tag, String.valueOf(value));
        }

    }

    public static void logW(String tag, long value) {
        if (!isDebug) {
            return;
        } else {
            Log.w(tag, String.valueOf(value));
        }
    }
}

File: app/src/main/java/com/shen/somebase/util/DebugLog.kt
package com.shen.somebase.util

import android.util.Log

/**
 * Created by hongzhang on 2018/4/10.
 */

object DebugLog {
    var isDebug = true

    fun logE(tag: String, value: String) {
        if (!isDebug) {
            return
        } else {
            Log.e(tag, value)
        }

    }

    fun logD(tag: String, value: String) {
        if (!isDebug) {
            return

        } else {
            Log.d(tag, value)
        }

    }

    fun logI(tag: String, value: String) {
        if (!isDebug) {
            return
        } else {
            Log.i(tag, value)
        }

    }

    fun logW(tag: String, value: String) {
        if (!isDebug) {
            return
        } else {
            Log.w(tag, value)
        }

    }
}
